docs/documentation/installation-guide.md

```diff
@@ -1,4 +1,143 @@
 # Installation guide
 
 !!! warning
-    **This section is under construction and should not be considered as accurate yet.**
\ No newline at end of file
+    **This section is under construction and should not be considered as accurate yet.**
+
+Here you can find the steps to follow to install the `Dimlpfidex` project on your machine. It can be archived on `Windows`, `Linux`, and `MacOS`. 
```

Comment:
Question: what is the proper capitalization of Dimlpfidex? Dimlpfidex? DimlpFidex? dimlpfidex? It's written as "dimlpfidex" in the README and doc, so we should aim for consistency (can be any, but aim for consistency).

Reply:
MacOS should be capitalized as "macOS", since it is an official brand.

Reply:
> Question: what is the proper capitalization of Dimlpfidex? Dimlpfidex? DimlpFidex? dimlpfidex? It's written as "dimlpfidex" in the README and doc, so we should aim for consistency (can be any, but aim for consistency).

That's a good question, let's say `dimlpfidex` for now.